dist/cms.js

```diff
@@ -151,12 +151,57 @@ function getWordPressContentType() {
   return content;
 }
 
+/**
+ * Obtains data about Interactivity API usage on the page.
+ *
+ * @returns {object} Object with fields `total_regions` and `total_regions_by_namespace`.
+ */
+function getInteractivityAPIUsage() {
+  // Look for data-wp-interactive regions.
+  const interactiveRegions = document.querySelectorAll('[data-wp-interactive]');
+
+  // Count the regions by namespace.
+  const regionNamespaceCounts = [];
```

Comment:
Should this be an object? So you get names for each subarray? Which will be useful when stringifying it to save in BigQuery:

```suggestion
  const regionNamespaceCounts = {};
```

That, coupled with the change below to `JSON.Stringify` leads to this on your sample page:

```'{"total_regions":6,"total_regions_by_namespace":{"core/navigation":6}}'```